src/bp-integrations/compatibility/bp-compatibility-integration.php

```diff
@@ -53,7 +53,7 @@ public function action_links( $link, $file ) {
 		$htmlDom = new DOMDocument;
 
 		// Parse the HTML of the page using DOMDocument::loadHTML
-		$htmlDom->loadHTML($link['settings']);
+		@$htmlDom->loadHTML($link['settings']);
```

Comment:
@shohelbuddyboss we don't want to just silence the errors. we want to fix them so that it doesn't come back again.